### PR TITLE
Vendors fixes

### PR DESCRIPTION
**Pull request by Humonitarian**
## About The Pull Request

Drinks Showcase showcases drinks now
Theomat no longer eats cash
Custom vendors no longer automatically guild-affiliated
MiniPharma finally shut up

## Why It's Good For The Game

Bugfix good

## Changelog
:cl:
fix: various vending machine issues
/:cl:

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1504,12 +1511,8 @@
...
 	return FALSE
 
-/obj/machinery/vending/theomat/vend(datum/data/vending_product/R, mob/user)
-	if(check_NT(user))
-		..()
-
-/obj/machinery/vending/theomat/try_to_buy(obj/item/W, var/datum/data/vending_product/R, var/mob/user)
-	if(check_NT(user))
+/obj/machinery/vending/theomat/attackby(obj/item/I, mob/user)
+	if(!I.tool_qualities && check_NT(user))
```
Comment (SirRichardFrancis) [inline code/game/machinery/vending.dm:1515]: Can only use non-tool and only if NT disciple? Why?

Comment (Humonitarian) [reply code/game/machinery/vending.dm:1515]: Ah good catch, those two should be separate.

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -1749,6 +1752,7 @@
 	name = "Custom Vendomat"
 	desc = "A custom vending machine."
 	custom_vendor = TRUE
+	vendor_department = null
```
Comment (TheShown911) [inline code/game/machinery/vending.dm:1755]: Sure this won't runtime without any department?

Comment (Humonitarian) [reply code/game/machinery/vending.dm:1755]: This is what they were before I set the default to guild

**Pull request merged**